vaadin-grid-flow/pom.xml

```diff
@@ -81,12 +81,12 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-checkbox</artifactId>
-            <version>2.2.8</version>
+            <version>2.2.10</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-text-field</artifactId>
-            <version>2.4.4</version>
+            <version>2.4.6</version>
```

Comment:
Should we use `vaadin-text-field` 2.4.7?

Reply:
Done.